identity/identity.go

```diff
@@ -132,36 +127,6 @@ type Identity struct {
 	NID       uuid.UUID `json:"-"  faker:"-" db:"nid"`
 }
 
-func (i *Identity) AfterEagerFind(tx *pop.Connection) error {
-	if err := i.setCredentials(tx); err != nil {
-		return err
-	}
-
-	if err := i.Validate(); err != nil {
-		return err
-	}
-
-	return UpgradeCredentials(i)
-}
```

Comment:
Does it hurt to keep this validation? Do we need to remove it? As far as I can tell, listing identities would no longer validate them nor upgrade the credentials, which might cause regressions down the road.

Reply:
Added validation+upgrade credentials to ListIdentities